### ISSUE TITLE
Missing 2 required positional arguments.

### ISSUE THREAD

**Issue by Lothedr** [tensorflow/datasets#2191]
**Short description**
Calling `tfds.builder('mnist')` causes exception:

> TypeError: \_\_init__() missing 2 required positional arguments: 'op' and 'message'

**Environment information**
* Operating System: Windows 10 Pro
* Python version: 3.8.3 (default, Jul  2 2020, 17:30:36) [MSC v.1916 64 bit (AMD64)]
* `tensorflow-datasets` version: 3.2.0
* `tensorflow` version: 2.2.0

**Reproduction instructions**

```
import tensorflow as tf
import tensorflow_datasets as tfds
builder = tfds.builder('mnist')
```

**Link to logs**
[Traceback](https://gist.github.com/Lothedr/5e7d835cb471d001c8f4b29f86085a8d)

**Expected behavior**
I expected that tfds will not throw an exception.

**Additional context**
There seems to be an implementation error [here](https://github.com/tensorflow/datasets/blob/a25626d3fd20c98a3ac99589cf68079022f2b232/tensorflow_datasets/core/utils/py_utils.py#L392). In my case, `exc_type` is `UnimplmentedError` and its constructor needs more arguments according to the [implementation](https://github.com/tensorflow/tensorflow/blob/909c073034e9b185149717c22b83d52dbd7396c3/tensorflow/python/framework/errors_impl.py#L436).

**Comment by vijayphoenix** [comment]
@Conchylicultor 
Minimal code to reproduce the issue is
```
import tensorflow as tf
tf.io.gfile.exists("gs://tfds-data/dataset_info/mnist/3.0.1")
```
If I remember correctly, TensorFlow has does not support for Google Cloud Storage (GCS) on Windows operating systems.

However, I already have commented on the existing TF issue https://github.com/tensorflow/tensorflow/issues/38477#issuecomment-657771830

**Comment by Conchylicultor** [comment]
Sorry for the breakage, it was introduced by using `tf.io.gfile` instead of `request` when we released `3.2.0`. In the meantime you can downgrade to `3.1.0`.

Marking this as duplicate of https://github.com/tensorflow/datasets/issues/2190
